src/app/services/user.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import {User} from "../user.model";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
```

Comment:
This class is very similar to UserService.

To provide some help with such repeating code, an abstract `CrudService` could be created with basic CRUD operations.

I don't really see this as utterly necessary and can be done in a future PR if wanted.